package.json

```diff
@@ -6,13 +6,14 @@
     "@testing-library/jest-dom": "^5.11.10",
     "@testing-library/react": "^11.2.5",
     "@testing-library/user-event": "^12.8.3",
+    "express": "^4.17.1",
```

Comment:
**[Optional]**
I am wondering if you will be adding a back-end to the application?? it is not a requirement, but I don't know how far you want to go personally on the project.

Reply:
Hi Forison,
I added express because I thought it was necessary to have a server to let Heroku build the app with node(server/server.js). Is it not necessary to deploy the app?  Can I achieve this just with the build create-react-app script?